data/transition-sites/bis_ukaea.yml

```diff
@@ -1,9 +1,10 @@
 ---
 site: bis_ukaea
 whitehall_slug: uk-atomic-energy-authority
+organisation_content_id: bb4afa19-6b2f-4bf7-9725-22a259c90c73
 host: www.uk-atomic-energy.org.uk
 tna_timestamp: 20130105231830
 homepage_furl: www.gov.uk/ukaea
 homepage: https://www.gov.uk/government/organisations/uk-atomic-energy-authority
-options:
+options: 
```

Comment:
This line adds trailing whitespace; `options` probably should be omitted from this file since it has no value.